app/assets/javascripts/service_request.coffee

```diff
@@ -120,14 +125,15 @@ $(document).on 'change, click', '.subcategories-wrapper input[type=radio]', ->
 setEquipment = ->
   location_id = $('#service_request_location_id').val()
   subcategory_id = $('.subcategories-wrapper .subcategory_field').val()
-  if location_id != ''
+  if location_id != undefined
```

Comment:
It's not correct check for undefined, you need to use `typeof location_id != 'undefined'`